src/Compilers/CSharp/Portable/Binder/Imports.cs

```diff
@@ -217,6 +217,8 @@ internal string GetDebuggerDisplay()
                                 }
                                 else
                                 {
+                                    Binder.ReportDiagnosticsIfObsoleteInternal(diagnostics, importedType, usingDirective.Name, importedType.ContainingSymbol, BinderFlags.None);
```

Comment:
I think the difference is that there is an explicit reference to the alias when it is use, this is not the case here. Also, looks consistent with how VB behaves.